src/marqo/index.py

```diff
@@ -282,7 +289,8 @@ def search(self, q: Optional[Union[str, dict]] = None, searchable_attributes: Op
             "reRanker": reranker,
             "boost": boost,
             "textQueryPrefix": text_query_prefix,
-            "hybridParameters": hybrid_parameters
+            "hybridParameters": hybrid_parameters,
+            "colbertFields": colbert_fields
```

Comment:
We can't release this field yet

Reply:
Ah yet. I messed up my mainline. Fix it now

Reply:
done